assets/bar.svg

```diff
@@ -1,24 +1,34 @@
-        <polygon points="356.283 66 380.283 66 380.283 255 356.283 255"/>
-        <polygon points="559.92 229 583.92 229 583.92 255 559.92 255"/>
-        <polygon points="101.738 115 125.738 115 125.738 255 101.738 255"/>
-        <polygon points="305.374 42 329.374 42 329.374 255 305.374 255"/>
-        <polygon points="509.011 201 533.011 201 533.011 255 509.011 255"/>
-        <polygon points="203.556 19 227.556 19 227.556 255 203.556 255"/>
-        <polygon points="407.192 115 431.192 115 431.192 255 407.192 255"/>
-      </g>
+<svg class="loading-state bar-load-state--css-wave" xmlns="http://www.w3.org/2000/svg" width="711" height="325" viewBox="0 0 711 325">
+    <defs>
+        <linearGradient id="lgrad" x1="0%" y1="50%" x2="100%" y2="50%" > 
+            <stop offset="0" stop-color="#ffffff" stop-opacity="0.8" />
+            <stop offset="33.33%" stop-color="#ffffff" stop-opacity="0.8" />
+            <stop offset="50%" stop-color="#ffffff" stop-opacity="0" />
+            <stop offset="66.66%" stop-color="#ffffff" stop-opacity="0.8" />
+            <stop offset="100%" stop-color="#ffffff" stop-opacity="0.8" />
+        </linearGradient>
+    </defs>
+    <g fill="none" fill-rule="evenodd">
+        <g transform="translate(0 29)">
+            <g stroke="#EFF2F5" stroke-dasharray="4 4">
+                <path d="M.400592911 200.302477L710.674315 200.302477M.400592884 131.172748L710.674315 131.172748M.324410282 64.2071321L710.621499 64.2071321M.291004517.563888874L709.82431.563888889"/>
+            </g>
+            <g fill="#D2D6DF" transform="translate(63.08 11)">
+                <polygon points="-.08 176 23.92 176 23.92 255 -.08 255"/>
+                <polygon points="50.829 147 74.829 147 74.829 255 50.829 255"/>
+                <polygon points="254.465 0 278.465 0 278.465 255 254.465 255"/>
+                <polygon points="458.102 169 482.102 169 482.102 255 458.102 255"/>
+                <polygon points="152.647 82 176.647 82 176.647 255 152.647 255"/>
+                <polygon points="356.283 66 380.283 66 380.283 255 356.283 255"/>
+                <polygon points="559.92 229 583.92 229 583.92 255 559.92 255"/>
+                <polygon points="101.738 115 125.738 115 125.738 255 101.738 255"/>
+                <polygon points="305.374 42 329.374 42 329.374 255 305.374 255"/>
+                <polygon points="509.011 201 533.011 201 533.011 255 509.011 255"/>
+                <polygon points="203.556 19 227.556 19 227.556 255 203.556 255"/>
+                <polygon points="407.192 115 431.192 115 431.192 255 407.192 255"/>
+            </g>
+        </g>
+        <polygon fill="#D2D6DF" fill-rule="nonzero" points="0 295 711 295 711 294 0 294"/>
     </g>
-    <polygon fill="#D2D6DF" fill-rule="nonzero" points="0 295 711 295 711 294 0 294"/>
-  </g>
-</svg>
+    <rect class="chart-filter" fill="url(#lgrad)" x="0" y="0" width="300%" height="100%"></rect>
+</svg>
```

Comment:
What do you think if we move these to /src/loadingStates ?
This assets/ folder was a temporal thing

Reply:
Ah, ok. Then do we even need these? I was considering deleting them.

Reply:
Ah, true, you are using the helper.
Go ahead and nuke all of it!